src/main/java/com/wanderersoftherift/wotr/item/riftkey/RiftKey.java

```diff
@@ -116,6 +116,16 @@ public void appendHoverText(
                         .withColor(ChatFormatting.GRAY.getColor()));
             }
         }
+        var parameters = stack.get(WotrDataComponentType.RiftKeyData.RIFT_PARAMETERS);
```

Comment:
Not sure rift keys need parameters, rather than have them derived from other data components... although since we don't have a modifier system just having direct parameters is a good way to play with them. Would suggest having a `RiftParameterProvider` interface which data components can implement to provide them - then a RiftTier component could provide the base values, a RiftTheme component could provide modifiers, etc. And then could sum them all from that interface. Similar to the ModifierProvider interface.